test/powershell/Modules/Microsoft.PowerShell.Management/Get-Item.Tests.ps1

```diff
@@ -17,7 +25,89 @@ Describe "Get-Item" -Tags "CI" {
         $childitem | Should BeOfType 'System.IO.FileInfo'
     }
 
+    It "Using -literalpath should find no additional files" {
+        $null = New-Item -type file "$TESTDRIVE/file[abc].txt"
+        $null = New-Item -type file "$TESTDRIVE/filea.txt"
+        # if literalpath is not correct we would see filea.txt
+        $item = Get-Item -literalpath "$TESTDRIVE/file[abc].txt"
+        @($item).Count | Should Be 1
+        $item.Name | Should Be 'file[abc].txt'
+    }
+
     It "Should have mode flags set" {
         Get-ChildItem $PSScriptRoot | foreach-object { $_.Mode | Should Not BeNullOrEmpty }
     }
+
+    It "Should not return the item unless force is used if hidden" {
+        ${hiddenFile} = "${TESTDRIVE}/.hidden.txt"
+        ${item} = New-Item -type file "${hiddenFile}"
+        if ( ${IsWindows} ) {
+            attrib +h "$hiddenFile"
+        }
+        ${result} = Get-Item "${hiddenFile}" -ErrorAction SilentlyContinue
+        ${result} | Should BeNullOrEmpty
+        ${result} = Get-Item -force "${hiddenFile}" -ErrorAction SilentlyContinue
+        ${result}.FullName | Should Be ${item}.FullName
+    }
+
+    Context "Test for Include, Exclude, and Filter" {
+        BeforeAll {
+            ${testBaseDir} = "${TESTDRIVE}/IncludeExclude"
+            $null = New-Item -Type Directory "${testBaseDir}"
+            $null = New-Item -Type File "${testBaseDir}/file1.txt"
+            $null = New-Item -Type File "${testBaseDir}/file2.txt"
+        }
+        It "Should respect -Exclude" {
+            $result = Get-Item "${testBaseDir}/*" -Exclude "file2.txt"
+            ($result).Count | Should Be 1
+            $result.Name | should be "file1.txt"
+        }
+        It "Should respect -Include" {
+            $result = Get-Item "${testBaseDir}/*" -Include "file2.txt"
+            ($result).Count | Should Be 1
+            $result.Name | should be "file2.txt"
+        }
+        It "Should respect -Filter" {
+            $result = Get-Item "${testBaseDir}/*" -Filter "*2*"
+            ($result).Count | Should Be 1
+            $result.Name | should be "file2.txt"
+        }
+    }
+
+    Context "Error Condition Checking" {
+        It "Should return an error if the provider does not exist" {
+            { Get-Item BadProvider::/BadFile -ErrorAction Stop } | ShouldBeErrorId "ProviderNotFound,Microsoft.PowerShell.Commands.GetItemCommand"
+        }
+
+        It "Should return an error if the drive does not exist" {
```

Comment:
i would rather get this in as is and add it later - it adds a very small additional percentage to coverage